docs/src/ci.md

```diff
@@ -587,6 +587,19 @@ tests:
     - npm ci
     - npx playwright test --project=$PROJECT --shard=$SHARD
 ```
+### Google Cloud Build
+* langs: js
+
+To run Playwright tests on Google Cloud Build, use our public Docker image ([see Dockerfile](./docker.md)).
+
+```yml
+steps:
+- name: mcr.microsoft.com/playwright:v%%VERSION%%-jammy
+  script: 
+  ...
+  env:
+  - 'CI=true'
```

Comment:
Does Google Clould CI not set it by default? I thought all modern cloud CI providers run with `CI=1`, we don't have it any other example.

Reply:
From what I have experienced it does not set it by default. I had to set it to trigger the CI rules in playwright config